src/geo/gmaps/gmaps-cartodb-layer-group-view.js

```diff
@@ -1,7 +1,7 @@
 /* global Image, google */
 var _ = require('underscore');
 var GMapsLayerView = require('./gmaps-layer-view');
-var zera = require('../../../vendor/zera');
+var zera = require('iagolast-zera');
```

Comment:
have you tried linking builder, di.js and carto ? is builder compilation working?

Reply:
The idea is to merge this working PR and then update `wax-webpack` branch in the integration with Builder & DI.